EMULATOR/EMULATOR.C

```diff
@@ -364,14 +366,14 @@ char* Emulator_GenerateColourArrayQuad(unsigned char* col0, unsigned char* col1,
 			vertices[0].col[0] = ((float)col0[0] * VERTEX_COLOUR_MULT) / 255.0f;
 			vertices[0].col[1] = ((float)col0[1] * VERTEX_COLOUR_MULT) / 255.0f;
 			vertices[0].col[2] = ((float)col0[2] * VERTEX_COLOUR_MULT) / 255.0f;
-			vertices[0].col[3] = ((float)0xFF) / 255.0f;
+			vertices[0].col[3] = ((float)INT8_MAX) / 255.0f;
```

Comment:
Oddly the outcome of this AFAIK is 1. If it will always be 1, I'm unsure why we would use this notation to get there?

Reply:
http://cpp.sh/8rzgs

Reply:
Because this is clear byte to float conversion formula.

Reply:
I mean why not just say it = 1.0f?

Reply:
Because the alpha component can change, It's currently not implemented in the emulator and is placeholder. 1.0f is the outcome of this operation for now but it may change in the future.

Reply:
Hey, I just  randomly looked at the PRs for this project.

This change is incorrect, the `INT8_MAX` is 127 and so 0x7f hexadecimally. This can also be seen on my fast checking it via an instance of geordi eval bot on irc:
```
02:35 <@disconnect3d> cxx << ((float)INT8_MAX) / 255.0f
02:35 <cxx> 0.498039
02:35 <@disconnect3d> cxx << ((float)UINT8_MAX) / 255.0f
02:35 <cxx> 1
02:35 <@disconnect3d> cxx << ((float)0xff) / 255.0f
02:35 <cxx> 1
```

If you still want to use `0xff`, use a proper constant of `UINT8_MAX`:
```suggestion
			vertices[0].col[3] = ((float)UINT8_MAX) / 255.0f;
```

*Note that the same issue can be observed in next lines (374, 384, 391, 402, 409, 419, 426, 437, ...).

Reply:
Okay, pointed by a friend (thx wiechu), the `INT8_MAX` maps to `0xff` here as it is added in this PR into emulator.h: https://github.com/TOMB5/TOMB5/pull/171/files#diff-e3a4032c69912aa62af6a2ead9fe4071R33

Anyway, the `UINT` vs `INT` issue should be fixed here.

Reply:
I'm not even certain these changes are a good idea which is why I paused the work